web/src/beta/components/SettingsButtons/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React, { ReactNode } from "react";
+
+import SettingsButtons from ".";
+
+export default {
+  component: SettingsButtons,
+} as Meta;
+
+type Story = StoryObj<typeof SettingsButtons>;
+
+const Wrapper: React.FC<{ children: ReactNode }> = ({ children }) => (
+  <div style={{ width: "96px" }}>{children}</div>
```

Comment:
Could you delete this? or please use `display: flex` instead of specifying `width`.
This component actually can handle long text, but it's broken by this `width`.
![image](https://github.com/reearth/reearth/assets/16318727/a0217c69-5aa2-4536-912c-1e0449803be4)